presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -101,6 +104,10 @@
     protected final String connectionUrl;
     protected final Properties connectionProperties;
     protected final String identifierQuote;
+    private final boolean mapTableNames;
+
+    private Map<String, Map<String, String>> schemaTableMapping = new HashMap<>();
```

Comment:
I was using what was there for the existing tests. Also was other wanting to end up with external applications needing to be run. You're referring to Postgres? In a quick search I actually see some Java libs for embedding Postgres and MySQL for testing purposes, I'll look into both if them.